src/components/profile-youtube-playlist/profile-youtube-playlist.js

```diff
@@ -0,0 +1,91 @@
+import { Helmet } from 'react-helmet';
+import React, { useState, useEffect } from 'react';
+import Carousel from '@brainhubeu/react-carousel';
+import '@brainhubeu/react-carousel/lib/style.css';
+import PropTypes from 'prop-types';
+
+const YoutubePlaylist = ({ youtubePlayListId }) => {
```

Comment:
I think the component filename should match the component name.

@christianmorfordwaitessw do you agree?

Reply:
Done - Renamed folder and file to match component name